src/libraries/System.Text.RegularExpressions/src/Resources/Strings.resx

```diff
@@ -261,6 +261,10 @@
     <value>RegexOptions.NonBacktracking is not supported in conjunction with expressions containing: '{0}'.</value>
     <comment>{Locked="RegexOptions.NonBacktracking"}</comment>
   </data>
+  <data name="NotSupported_NonBacktrackingUnsafeSize" xml:space="preserve">
+    <value>The specified pattern with RegexOptions.NonBacktracking could result in an automata as large as '{0}' nodes, which is larger than the default limit of '{1}'.</value>
```

Comment:
Should we change the word `default` for the word `configured` instead? That way if someone is using the environment variable to override the default but that value is also exceeded, then the message would still be correct.

Reply:
I agree, the **default** is actually 1000, while this could indeed be the **configured** case in general. But I have no strong opinion on this rewording.

Reply:
If no one objects, I would actually go ahead and change **default** to **configured**.